src/ansys/grantami/recordlists/_connection.py

```diff
@@ -43,7 +43,7 @@
 API_DEFINITION_PATH = "/swagger/v1/swagger.json"
 GRANTA_APPLICATION_NAME_HEADER = "PyGranta RecordLists"
 
-MINIMUM_GRANTA_MI_VERSION = (24, 2)
+MINIMUM_GRANTA_MI_VERSION = (25, 2)
```

Comment:
This may change serverapi-openapi can include multiple versions of the API bindings. But this won't be introduced in this PR.